### PR TITLE
Add build failure reasons to monitoring and overview pages

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett missed this two pages, i couldn't decide between the long or short form on the overview

File: app/views/monitoring.html [hunk 1]
```diff
@@ -303,7 +303,9 @@
                               <small>created <span am-time-ago="build.metadata.creationTimestamp"></span></small>
                             </div>
                             <div class="list-group-item-text">
-                              <build-status build="build"></build-status>
```
Comment (spadgett) [inline app/views/monitoring.html]: Should we delete the build-status directive if it's not used? Or update it to show the reason or message instead?

**Comment by jwforres** [comment]
Realized this change was removing the duration completely from the monitoring page, i'm updating build-status directive for when we still need status + duration, but using the longer form duration on this page there isn't really enough space for it when the events sidebar is expanded.

wrapping

**Comment by spadgett** [comment]
Yeah I think we need to use "1m 48s" format, maybe small / text-muted?

**Comment by jwforres** [comment]
i dunno about changing the font style, in some cases its read as part of a statement like with Cancelled

**Comment by jwforres** [comment]
updated PTAL

**Comment by spadgett** [comment]
[merge]

**Pull request merged**